typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -11,79 +11,49 @@ import { MultiProvider } from '../providers/MultiProvider.js';
 import { EthersV5Transaction } from '../providers/ProviderType.js';
 import { ChainNameOrId } from '../types.js';
 
-import { EvmIsmCreator } from './EvmIsmCreator.js';
-import { EvmIsmReader } from './EvmIsmReader.js';
+import { DerivedIsmConfigWithAddress, EvmIsmReader } from './EvmIsmReader.js';
 import { IsmConfig } from './types.js';
 
 export class EvmIsmModule extends HyperlaneModule<
   ProtocolType.Ethereum,
   IsmConfig,
-  HyperlaneContracts<ProxyFactoryFactories> & {
+  HyperlaneAddresses<ProxyFactoryFactories> & {
     deployedIsm: Address;
   }
 > {
   protected logger = rootLogger.child({ module: 'EvmIsmModule' });
   protected reader: EvmIsmReader;
-  protected creator: EvmIsmCreator;
 
   protected constructor(
     protected readonly multiProvider: MultiProvider,
     protected readonly deployer: HyperlaneDeployer<any, any>,
     args: HyperlaneModuleArgs<
       IsmConfig,
-      HyperlaneContracts<ProxyFactoryFactories> & {
+      HyperlaneAddresses<ProxyFactoryFactories> & {
         deployedIsm: Address;
       }
     >,
   ) {
     super(args);
     this.reader = new EvmIsmReader(multiProvider, args.chain);
-    this.creator = new EvmIsmCreator(deployer, multiProvider, args.addresses);
   }
 
-  public async read(): Promise<IsmConfig> {
+  public async read(): Promise<DerivedIsmConfigWithAddress> {
     return await this.reader.deriveIsmConfig(this.args.addresses.deployedIsm);
   }
 
-  public async update(config: IsmConfig): Promise<EthersV5Transaction[]> {
+  public async update(_config: IsmConfig): Promise<EthersV5Transaction[]> {
```

Comment:
Ignore these IsmModule changes since @paulbalaji PR will implement

Reply:
it seems like you're deleting what will be dead code - so maybe ignore isn't right, but just clarifying these methods will be implemented after a followup PR lands?

Reply:
Yeah these files were passed over by @paulbalaji. They're used like interfaces, so I won't be impacted by his PR. 

The idea is that my PR should land first, then his ISM implementations